.gitignore

```diff
@@ -10,3 +10,4 @@ anvio/data/misc/COG/
 anvio/data/misc/MODELLER/db/
 smtp_config.ini
 .snakemake
+anvio/data/misc/
```

Comment:
don't add the entire directory to be ignored, be specific like this one in the same file:

```
anvio/data/misc/MODELLER/db/
```